pkg/test/resourcefixture/testdata/basic/resourcemanager/v1beta1/project/projectmovedfoldertofolder/_generated_object_projectmovedfoldertofolder.golden.yaml

```diff
@@ -24,5 +24,5 @@ status:
     reason: UpToDate
     status: "True"
     type: Ready
-  number: "2589592254"
+  number: "518915279"
```

Comment:
This isn't a magic number, is it?

Reply:
not ready for review. I have a better idea.